components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -153,7 +156,8 @@ public Application getApplicationByName(String clientName) throws DCRMException
      * @return
      * @throws DCRMException
      */
-    public Application registerApplication(ApplicationRegistrationRequest registrationRequest) throws DCRMException {
+    public Application registerApplication(ApplicationRegistrationRequest registrationRequest) throws DCRMException,
```

Comment:
Adding an additional exception type to a method signature will lead to failures of existing methods which use this method.